tests/integration/cli/src/assets.rs

```diff
@@ -1,51 +1,70 @@
-use std::env;
 use std::path::PathBuf;
+use std::{env, path::Path};
 
-pub const C_ASSET_PATH: &str = concat!(
-    env!("CARGO_MANIFEST_DIR"),
-    "/../../../lib/c-api/examples/assets/"
-);
-pub const ASSET_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/../../../tests/examples/");
+pub fn c_asset_path() -> &'static Path {
+    Path::new(concat!(
+        env!("CARGO_MANIFEST_DIR"),
+        "/../../../lib/c-api/examples/assets/"
+    ))
+}
 
-pub const WASMER_INCLUDE_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/../../../lib/c-api/");
+pub fn asset_path() -> &'static Path {
+    Path::new(concat!(
+        env!("CARGO_MANIFEST_DIR"),
+        "/../../../tests/examples/"
+    ))
+}
 
-#[cfg(feature = "debug")]
-pub const WASMER_TARGET_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/../../../target/debug/");
-#[cfg(feature = "debug")]
-pub const WASMER_TARGET_PATH_2: &str = concat!(
-    env!("CARGO_MANIFEST_DIR"),
-    "/../../../target/",
-    env!("CARGO_BUILD_TARGET"),
-    "/debug/"
-);
+pub fn wasmer_include_path() -> &'static Path {
+    Path::new(concat!(env!("CARGO_MANIFEST_DIR"), "/../../../lib/c-api/"))
+}
 
-/* env var TARGET is set by tests/integration/cli/build.rs on compile-time */
+pub fn wasmer_target_path() -> &'static Path {
+    let path = if cfg!(feature = "debug") {
+        concat!(env!("CARGO_MANIFEST_DIR"), "/../../../target/debug/")
```

Comment:
When building from the CLI, the path might be `target/x86_64-linux/release` or something similar as we define the target arch even when not cross-compiling